react/react1/week2/todo-list/src/components/ToDoList.jsx

```diff
@@ -0,0 +1,85 @@
+import TaskItem from "./TaskItem";
+import tasklist from "../data/tasklist";
+import { useState } from "react";
+
+export default function ToDoList() {
+  const setTaskList = () => {
+    return tasklist.map((item) => ({ ...item, isChecked: false }));
+  };
+
+  const [items, setItems] = useState(() => setTaskList());
```

Comment:
You can use `useState(tasks);` .

Reply:
@Divyajg Thanks a lot for review.  I will fix the things you pointed out.